napari/_qt/qt_about_keybindings.py

```diff
@@ -1,105 +1,80 @@
 from ..util.misc import get_keybindings_summary
 
 
-class QtAboutKeybindings(QTabWidget):
-    def __init__(self, viewer, parent):
-        super(QtAboutKeybindings, self).__init__(parent)
-
-        self.viewer = viewer
-
-        self.addTab(QtActiveKeybindings(self.viewer), 'Currently active')
-        self.addTab(QtLayerKeybindings(napari.layers.Image), 'Image')
-        self.addTab(QtLayerKeybindings(napari.layers.Labels), 'Labels')
-        self.addTab(QtLayerKeybindings(napari.layers.Points), 'Labels')
-        self.addTab(QtLayerKeybindings(napari.layers.Shapes), 'Shapes')
-        self.addTab(QtLayerKeybindings(napari.layers.Surface), 'Surface')
-        self.addTab(QtLayerKeybindings(napari.layers.Vectors), 'Vectors')
-
-    @staticmethod
-    def showAbout(qt_viewer):
-        d = QDialog()
-        d.setObjectName('QtAboutKeybindings')
-        d.setStyleSheet(qt_viewer.styleSheet())
-        d.setWindowTitle('Keybindings')
-        qt_viewer._about_keybindings = QtAboutKeybindings(qt_viewer.viewer, d)
-        d.show()
-        d.setWindowModality(Qt.NonModal)
-        d.setSizePolicy(QSizePolicy.Expanding, QSizePolicy.Expanding)
-        qt_viewer._about_keybindings_dialog = d
-
-
-class QtActiveKeybindings(QScrollArea):
+class QtAboutKeybindings(QWidget):
     def __init__(self, viewer):
         super().__init__()
 
         self.viewer = viewer
-        self.setWidgetResizable(True)
-        self.setHorizontalScrollBarPolicy(Qt.ScrollBarAlwaysOff)
-
-        self.active_label = QLabel()
-        self.active_label.setTextInteractionFlags(Qt.TextSelectableByMouse)
-        self.active_label.setAlignment(Qt.AlignLeft)
-        self.active_label.setContentsMargins(10, 10, 10, 10)
-        self.active_label.setSizePolicy(
-            QSizePolicy.Expanding, QSizePolicy.Expanding
-        )
-
-        self.update_text(None)
+        self.layout = QVBoxLayout()
+        self.setLayout(self.layout)
 
-        scroll_widget = QWidget()
-        scroll_widget.setSizePolicy(
-            QSizePolicy.Expanding, QSizePolicy.Expanding
+        # stacked keybindings widgets
+        self.textEditBox = QTextEdit()
+        self.textEditBox.setTextInteractionFlags(Qt.TextSelectableByMouse)
+        self.textEditBox.setMinimumWidth(360)
+        self.keybindings_strs = {'All active keybindings': ''}
+        layers = [
+            napari.layers.Image,
+            napari.layers.Labels,
+            napari.layers.Points,
+            napari.layers.Shapes,
+            napari.layers.Surface,
+            napari.layers.Vectors,
+        ]
+        for layer in layers:
+            if len(layer.class_keymap) == 0:
+                text = 'No keybindings'
+            else:
+                text = get_keybindings_summary(layer.class_keymap)
+            self.keybindings_strs[f"{layer.__name__} layer"] = text
+
+        # layer type selection
+        self.layerTypeComboBox = QComboBox()
+        for name in self.keybindings_strs.keys():
```

Comment:
the `.keys()` part is implicit when iterating over a dict

Reply:
done